packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [28.1.1]
+
+### Uncategorized
```

Comment:
PR 4420 (referenced below) bumped `gas-fee-controller` in `transaction-controller` (since it was one of the packages that got released). So, we can file that under **Changed**:

``` suggestion
### Changed

- Bump `@metamask/gas-fee-controller` to ^15.1.0 ([#4220](https://github.com/MetaMask/core/pull/4220))

```

Reply:
Applied.